source/integration/references/compile.rst

```diff
@@ -76,17 +76,30 @@ where *<install\_options>* can be one or more of:
 | **-h** | prints installer help                                                                                                                                                        |
 +--------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+.. note:: 
+    
+    If you choose the ``system-wide`` installation, OpenNebula will be installed in the following folders: 
+        -   /etc/one 
+        -   /share/docs/one
+        -   /usr/share/one
+        -   /var/lib/one
+        -   /var/lock/one
+        -   /var/log/one
+        -   /var/run/one
+        
+    By using ``./install.sh -r``, dinamically generated files will not be removed.
+
 The packages do a ``system-wide`` installation. To create a similar environment, create a ``oneadmin`` user and group, and execute:
 
 .. prompt:: text $ auto
 
     oneadmin@frontend:~/ $> wget <opennebula tar gz>
     oneadmin@frontend:~/ $> tar xzf <opennebula tar gz>
-    oneadmin@frontend:~/ $> cd one-4.0
-    oneadmin@frontend:~/one-4.0/ $> scons -j2 mysql=yes syslog=yes
+    oneadmin@frontend:~/ $> cd opennebula-5.x.x
+    oneadmin@frontend:~/opennebula-5.x.x/ $> scons -j2 mysql=yes syslog=yes
```

Comment:
maybe instead of saying opennebula-5.x.x, we could do, opennebula-x.y.z so we don't need to change this line int he future